microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+* Your extension must not do anything that interferes with or diminishes the visibility, value, or quality of any ads that it does display.
 
-#### 1.10.4 Promotions
+
+###### 1.10.4 Promotions
 
 If you purchase or create promotional ad campaigns to promote your extensions through the ad campaign functionality in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), all ad materials you provide to Microsoft, including any associated landing pages, must comply with [Microsoft Creative Specifications Policy](https://about.ads.microsoft.com/policies/creative-specs) and [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
-#### 1.10.5 Notifying Users of Opt-Out for Interest-Based Advertising
+
+###### 1.10.5 Notifying users of opt-out for interest-based advertising
 
 Your privacy statement or terms of use must let users know you plan to send Personal Information to the ad service provider and must tell users how they may opt-out of interest-based advertising.
 
-#### 1.10.6 Other guidelines
+
+###### 1.10.6 Other guidelines
 
 If your extension is directed at children under the age of 13, as defined in the [Children's Online Privacy Protection Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/children%27s-privacy); you must notify Microsoft of this fact in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and ensure that all ad content displayed in your extension is appropriate for children under the age of 13.
 
 
 <!-- ====================================================================== -->
-## 2 Content Policies
+## 2 Content policies
+
+The following policies apply to content and metadata (including publisher name, extension name, extension icon, extension description, extension screenshots, extension trailers and trailer thumbnails, and any other extension metadata) offered for distribution in the Microsoft Edge Add-ons store.
+
+Content means the images, sounds, videos, and text contained in your extension, the tiles, notifications, error messages, or ads exposed through your extension, and anything delivered from a server or to which your extension connects.
+
+Because extensions and Microsoft Edge Add-ons are used around the world, these requirements are interpreted and applied in the context of regional and cultural norms
+
+
+<!-- ------------------------------ -->
+#### 2.1 Content requirements
+
+For your extension to be listed in the Microsoft Edge Add-ons store, your extension and metadata you submit should not contain mature, explicit, or otherwise inappropriate content.
+
+This includes content that is:
+* Discriminatory, hateful, or offensive.
+* Harmful or dangerous.
+* Misleading or fraudulent.
+* Spam or malware.
+
+Examples of specific types of content that would be considered inappropriate for Microsoft Edge Add-ons include:
```

Comment:
for Edge Add-Ons

Reply:
would be considered inappropriate for Edge Add-ons include: